apps/docs/pages/availability.mdx

```diff
@@ -21,7 +21,7 @@ This will tell you when your calendar shows you as busy, so you can understand w
 2. To the right of your screen a box saying 'Something doesn't look right?' will appear. Press the button 'Launch Troubleshooter'.
 3. After pressing this, your availability will appear according to your calendar and inform you of the times booked up for that day!
 
-By doing this you will know understand why certain times are available and others are blocked.
```

Comment:
@alishaz-polymath Should we remove the docs folder?

Reply:
was just thinking the same #9061